state/forcedoperation.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Canonical Ltd.
```

Comment:
This code could also move to modeloperation.go?  Does not belong in unit.go though.

Reply:
I think it is better here than modeloperation.go.